### PR TITLE
add sampling of check keys for one specific check

### PR DESCRIPTION
**Pull request by vetinari**
to do some stats on this

### CODE REVIEW

File: src/main/java/de/zalando/zmon/dataservice/data/KairosDBStore.java [hunk 1]
```diff
@@ -215,6 +216,9 @@ void store(WorkerResult wr) {
                 if (cdResultSize > resultSizeWarning) {
                     LOG.warn("result size warning: check={} data-points={} entity={} tags={}", cd.checkId, cdResultSize, cd.entityId, getEntityTags(cd.entity));
                 }
+                if (cd.checkId == config.getCheckMetricsWatchId() && Math.random() <= 0.1) {
+                    LOG.warn("sample for check={} {}", cd.checkId, values.keys())
```
Comment (alexkorotkikh) [inline src/main/java/de/zalando/zmon/dataservice/data/KairosDBStore.java]: why warning? shouldn't it be info?

Comment (vetinari) [reply src/main/java/de/zalando/zmon/dataservice/data/KairosDBStore.java]: done

**Comment by vetinari** [comment]
:+1:

**Comment by alexkorotkikh** [comment]
:+1:

**Pull request merged**